src/Microsoft.Extensions.Options/Options.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Microsoft.Extensions.Options
+{
+    public static class Options
+    {
+        public static IOptions<TOptions> Create<TOptions>(TOptions options) where TOptions : class, new()
+        {
+            return new Options<TOptions>(options);
+        }
+    }
+
+    public class Options<TOptions> : IOptions<TOptions> where TOptions : class, new()
```

Comment:
I think we should name this something less generic, as it currently stands `Options<TOptions>` might lead people to think this is the base class for options...

OptionsWrapper?  It won't be exposed really since users will almost always just use `Options.Create`?